examples/bpk-component-rating/examples.js

```diff
@@ -22,356 +22,158 @@ import React from 'react';
 import { cssModules } from '../../packages/bpk-react-utils';
 import BpkRating, {
   RATING_SIZES,
-  RATING_TYPES,
   RATING_SCALES,
 } from '../../packages/bpk-component-rating';
 
 import STYLES from './examples.module.scss';
 
 const getClassName = cssModules(STYLES);
 
-const DefaultExample = () => (
-  <div>
-    <BpkRating
-      ariaLabel="9 Excellent would recommend"
-      title="Excellent"
-      subtitle="This place was amazing"
-      value={9}
-    />
-    <br />
-    <BpkRating
-      ariaLabel="6.7 Average might recommend"
-      title="Average"
-      subtitle="Might recommend"
-      value={6.7}
-    />
-    <br />
-    <BpkRating
-      ariaLabel="2.3 Bad avoid here"
-      title="Bad"
-      subtitle="Avoid here"
-      value={2.3}
-    />
-  </div>
-);
-
-const ZeroToFiveScaleExample = () => (
-  <div>
-    <BpkRating
-      ariaLabel="4 Excellent would recommend"
-      title="Excellent"
-      subtitle="This place was amazing"
-      value={4.1}
-      ratingScale={RATING_SCALES.zeroToFive}
-    />
-    <br />
-    <BpkRating
-      ariaLabel="3 Average might recommend"
-      title="Average"
-      subtitle="Might recommend"
-      value={3.4}
-      ratingScale={RATING_SCALES.zeroToFive}
-    />
-    <br />
-    <BpkRating
-      ariaLabel="2 Bad avoid here"
-      title="Bad"
-      subtitle="Avoid here"
-      value={2.3}
-      ratingScale={RATING_SCALES.zeroToFive}
-    />
-  </div>
-);
-
-const Vertical = () => (
-  <div>
-    <BpkRating
-      className={getClassName('bpk-rating-story')}
-      ariaLabel="9 Excellent would recommend"
-      title="Excellent"
-      subtitle="This place was amazing"
-      value={9}
-      vertical
-    />
-    <br />
-    <BpkRating
-      className={getClassName('bpk-rating-story')}
-      ariaLabel="6.7 Average might recommend"
-      title="Average"
-      subtitle="Might recommend"
-      value={6.7}
-      vertical
-    />
-    <br />
-    <BpkRating
-      className={getClassName('bpk-rating-story')}
-      ariaLabel="2.3 Bad avoid here"
-      title="Bad"
-      subtitle="Avoid here"
-      value={2.3}
-      vertical
-    />
-  </div>
-);
-
-const LargeSize = () => (
-  <div>
-    <BpkRating
-      ariaLabel="9 Excellent would recommend"
-      title="Excellent"
-      subtitle="This place was amazing"
-      value={9}
-      size={RATING_SIZES.lg}
-    />
-    <br />
-    <BpkRating
-      ariaLabel="6.7 Average might recommend"
-      title="Average"
-      subtitle="Might recommend"
-      value={6.7}
-      size={RATING_SIZES.lg}
-    />
-    <br />
-    <BpkRating
-      ariaLabel="2.3 Bad avoid here"
-      title="Bad"
-      subtitle="Avoid here"
-      value={2.3}
-      size={RATING_SIZES.lg}
-    />
-  </div>
+const ImageTiltle = (
+  <img
+    alt="rating 4.5"
+    className={getClassName('bpk-rating-story--img')}
+    src="https://www.tripadvisor.com/img/cdsi/img2/ratings/traveler/4.5-64600-4.svg"
```

Comment:
You'll need to remove this and keep just the generic one on the examples so that we don't violate any copyright due to open source 

Either we need a logo from OUR cdn or I can provide a sample company logo if we want to show an example

Reply:
Thanks for your review and help pointing out copyright issue, i've replace them to generic and add one node example instead.

Reply:
Thanks for your review and help pointing out copyright issue, i've replace them to logo from OUR cdn.